src/js/tabs/osd.js

```diff
@@ -2401,30 +2403,38 @@ TABS.osd.initialize = function (callback) {
         // init structs once, also clears current font
         FONT.initData();
 
+        // Some of these definitions are determined by version.
+        SYM.AH_CENTER_LINE = 0x26;
+        SYM.AH_CENTER_LINE_RIGHT = 0x27;
+        if(semver.gte(CONFIG.apiVersion,"1.42.0")) {
+            SYM.AH_CENTER_LINE = 0x7B;
+            SYM.AH_CENTER_LINE_RIGHT = 0x7D;
+        }
+
         fontPresetsElement.change(function (e) {
             var $font = $('.fontpresets option:selected');
-            $.get('./resources/osd/' + $font.data('font-file') + '.mcm', function (data) {
+            var fontver = '1';
+            if (semver.gte(CONFIG.apiVersion,"1.42.0")) {
+                fontver = '2';
+            }
+            $('.font-manager-version-info').text(i18n.getMessage('osdDescribeFontVersion'+fontver));
```

Comment:
Please use space around operators (except for on the inside of braces).

Essentially, most of the rules in https://github.com/betaflight/betaflight/blob/master/docs/development/CodingStyle.md are applicable to JavaScript as well.